notebooks/224-stable-diffusion-text-to-image/224-stable-diffusion-text-to-image.ipynb

```diff
@@ -0,0 +1,1209 @@
+    "See the [model card](https://huggingface.co/CompVis/stable-diffusion) for more information.\n",
+    "\n",
+    "General diffusion models are machine learning systems that are trained to denoise random gaussian noise step by step, to get to a sample of interest, such as an image.\n",
+    "Diffusion models have shown to achieve state-of-the-art results for generating image data. But one downside of diffusion models is that the reverse denoising process is slow. In addition, these models consume a lot of memory because they operate in pixel space, which becomes unreasonably expensive when generating high-resolution images. Therefore, it is challenging to train these models and also use them for inference. OpenVINO brings capabilities to run model inference on Intel hardware and opens the door to the fantastic world of diffusion models for everyone!\n",
+    "\n",
+    "\n",
+    "This notebook demonstrates how to convert and run stable diffusion using OpenVINO.\n",
+    "\n",
+    "Notebook contains the following steps:\n",
+    "1. Convert PyTorch models to ONNX format.\n",
+    "2. Convert ONNX models to OpenVINO IR format using Model Optimizer tool.\n",
+    "3. Run Stable Diffusion pipeline with OpenVINO."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    ">**Note**:\n",
+    "For original model (e.g. `stable-diffusion-v1-4`) usage you also need to accept the model license before downloading or using the weights, visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree. \n",
+    "You have to be a registered user in 🤗 Hugging Face Hub, and you'll also need to use an access token for the code to work. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).\n",
+    "You can login on HuggingFace Hub in notebook environment using following code:\n",
+    "```python\n",
+    "## login to huggingfacehub to get access to pretrained model \n",
+    "from huggingface_hub import notebook_login, whoami\n",
+    "\n",
+    "try:\n",
+    "    whoami()\n",
+    "    print('Authorization token already provided')\n",
+    "except OSError:\n",
+    "    notebook_login()\n",
+    "```\n",
+    "\n",
+    "In this tutorial we will use Stable Diffusion fine tuned on Midjourney v4 images (another popular solution for text to image generation), you can find more details about this model in [model card](https://huggingface.co/prompthero/midjourney-v4-diffusion). The same steps for conversion and pipeline running are also applicable for other solutions based on Stable diffusion.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -r requirements.txt"
```

Comment:
I got your point with the installation option here.

Reply:
so, what is solution? should I stay my requirements installation  as is or put info about this step in readme?